### DISCUSSION TITLE
thanhle.blog/ki-su-freelancer-o-upwork

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#72] [Ideas]
# thanhle.blog/ki-su-freelancer-o-upwork

Hành trình làm freelancer từ rate 14$/hr tới Top rated freelancer với rate 45$/hr của mình

https://thanhle.blog/blog/ki-su-freelancer-o-upwork

**Comment by hoanghung123vn** [comment]
liked

**Comment by haka277** [comment]
Bài viết rất hay đó. Sau khi đọc 7749 bài review trên tiktok thì lượn qua đây thấy bài này chia sẻ chân thực này. Chứ k vẽ ra ảo tưởng như trên tóp tóp. Mình cũng đang trong giai đoạn chênh vênh khi các cơ quan đều cắt giảm nhân sự. Làm một cv nhà nước khá an nhàn và ổn định nhưng lại tẻ nhạt và k có sự phát triển. Nhưng mình khá băn khoăn vì công việc của mình không phải là chuyên môn như các bạn kỹ thuật mà làm về khối bổ trợ (hành chính, kế toán, nhân sự,...). Nên ở những khối này rất dễ bị cắt giảm và lúc nào cũng có thể thay thế. Vậy nếu làm việc trên upwork liệu có cv phù hợp cho mình k nhỉ?
